apps/vanilla/src/scripts/renderSortingAndOrdering.ts

```diff
@@ -0,0 +1,79 @@
+import { OrderOption } from '@js-camp/core/enum';
+import { AnimeListQueryOptions } from '@js-camp/core/models/animeListQueryOptions';
+import { Sorting } from '@js-camp/core/models/sorting';
+
+import { DEFAULT_LIMIT, SORT_OPTIONS } from '../constants';
+import { KEY_ORDER, KEY_SORTING } from '../constants/key';
+import { LocalStorageService } from '../service/localStorage';
+import { setDefaultSelected } from '../utils';
+
+import { renderListAndPaginationToUI } from './renderPagination';
+
+const selectSort = document.querySelector<HTMLSelectElement>('.filter__item_select-sort');
+const selectOrdering = document.querySelector<HTMLSelectElement>('.filter__item_select-order');
+
+/** Init and render sorting and ordering list, hence render corresponding list anime.
+ * @param options Options of pagination.
+ */
+export function renderSortingAndOrdering(options: AnimeListQueryOptions): void {
+  const sortOptionHTML = SORT_OPTIONS.map(item => (
+    `<option value="${item.title}">${item.title}</option>`
+  )).join('');
+
+  const orderOptionHTML = Object.keys(OrderOption).map(item => (
+    `<option value ="${item}">${item}</option>`
+  ))
+    .join('');
+  if (selectSort === null || selectSort === undefined) {
+    return;
+  }
+  selectSort.innerHTML = sortOptionHTML;
+  setDefaultSelected(selectSort, LocalStorageService.getValue<Sorting>(KEY_SORTING)?.title ?? SORT_OPTIONS[0].title);
+  selectSort.addEventListener('change', () => {
+      const { value } = selectSort;
+      LocalStorageService.setValue(KEY_SORTING, SORT_OPTIONS.filter(item => item.title === value)[0]);
+    const selectSortingValue = LocalStorageService.getValue<Sorting>(KEY_SORTING) ?? SORT_OPTIONS[0];
+
+    const optionsUpdated = new AnimeListQueryOptions({
+      ...options,
+      offset: DEFAULT_LIMIT,
+      activePage: 1,
+      sorting: new Sorting({
+        ...options.sorting,
+        ...selectSortingValue,
+      }),
+    });
+      renderListAndPaginationToUI(optionsUpdated);
+
+    });
+
+  if (selectOrdering === null || selectOrdering === undefined) {
```

Comment:
This would do the same
```suggestion
  if (selectOrdering == null) {
```